tests/profiling/collector/test_memalloc.py

```diff
@@ -220,7 +220,7 @@ def test_heap():
     x = _allocate_1k()
     # Check that at least one sample comes from the main thread
     thread_found = False
-    for (stack, nframe, thread_id), size in _memalloc.heap():
+    for (stack, _nframe, thread_id), size in _memalloc.heap():
```

Comment:
What is the problem here?

Reply:
Another instance of `B007: Loop control variable not used within the loop body. If this is intended, start the name with an underscore.`